client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -277,12 +276,12 @@ class RegistrantExtraInfoFrForm extends React.PureComponent {
 
 	renderOrganizationFields() {
 		const translate = this.props.translate;
-		const extra = this.props.contactDetails.extra;
+		const { extra } = this.props.contactDetails;
 		const {
 			registrantVatId,
 			sirenSiret,
-			trademarkNumber,
-		} = { ...emptyValues, ...extra };
+			trademarkNumber
+		} = defaults( extra, emptyValues );
```

Comment:
^ I accidentally submitted the above ahead of time. I meant to add that using `defaultProps` would have the added benefit (aside from less boilerplate, more declarative code) of only doing this work on component initialization, and not on each render.